api/Makefile

```diff
@@ -48,7 +48,27 @@ go_import_mapping_entries := \
 	google/protobuf/timestamp.proto=github.com/golang/protobuf/ptypes/timestamp \
 	google/protobuf/wrappers.proto=github.com/golang/protobuf/ptypes/wrappers \
 	envoy/api/v2/discovery.proto=github.com/envoyproxy/go-control-plane/envoy/api/v2 \
-	envoy/service/discovery/v3/discovery.proto=github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3
+	envoy/service/discovery/v3/discovery.proto=github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3 \
```

Comment:
Is this legacy from the gogo proto days? Are we sure that it is still needed?

Reply:
Yes, we need this. Envoy protos do not have go_package https://github.com/envoyproxy/envoy/issues/17082

Reply:
:-(

Reply:
I filed #2270